# FTP::Read
#
# Call FTP::Open first.

string file_path
uint64 offset
uint64 size
---
uint8[] data
bool success
